osbenchmark/builder/installers/bare_installer.py

```diff
@@ -0,0 +1,76 @@
+from osbenchmark.builder.installers.installer import Installer
+from osbenchmark.builder.installers.preparers.plugin_preparer import PluginPreparer
+from osbenchmark.builder.provision_config import BootstrapPhase
+from osbenchmark.builder.utils.config_applier import ConfigApplier
+from osbenchmark.builder.utils.java_home_resolver import JavaHomeResolver
+from osbenchmark.builder.utils.path_manager import PathManager
+from osbenchmark.builder.utils.template_renderer import TemplateRenderer
+
+
+class BareInstaller(Installer):
```

Comment:
Bare essentially means "not in a container". The original implementation used this name and I carried forward since I can't think of a better one. Open to suggestions if anyone has ideas